test/custom-matter-xml.test.js

```diff
@@ -456,45 +456,80 @@ test(
 test(
   'Add 2 custom files with code collisions',
   async () => {
+    // creating a new session with new uuid
+    let uuid = util.createUuid()
     conflictSid = await testQuery.createSession(
       db,
       'USER',
-      'SESSION',
+      uuid,
       env.builtinMatterZclMetafile()
     )
+
     let result = await zclLoader.loadIndividualFile(
       db,
       testUtil.testMattterCustomXml,
       conflictSid
     )
     expect(result.succeeded).toBeTruthy()
 
-    try {
-      result = await zclLoader.loadIndividualFile(
+    result = await zclLoader.loadIndividualFile(
+      db,
+      testUtil.testMatterConflict,
+      conflictSid
+    )
+    expect(result.succeeded).toBeTruthy()
+    let conflictPackageId = result.packageId
+
+    let sessionNotif = await querySessionNotification.getNotification(
+      db,
+      conflictSid
+    )
+    expect(
+      sessionNotif.some((notif) =>
+        notif.message.includes('Cluster code conflict')
```

Comment:
Can these messages also check for the path you inserted to be more precise on the error?